mobile/src/Student/screens/SelectTeam/index.js

```diff
@@ -66,7 +65,7 @@ const SelectTeam = ({ navigation, team, saveTeamAvatar }) => {
         <View style={styles.teamIconNameContainer}>
           <Image style={styles.largeIcon} source={avatar && avatar.largeSrc} />
           <Text style={styles.fullNameText}>
-            {team.name}
+            {team ? team.name: null}  {/*propertyuuid doesn't exist*/}
```

Comment:
I might be missing something here, but the comment is about uuid while this is using the name